hub/scripts.js

```diff
@@ -121,54 +160,88 @@ function handleMetadata() {
 }
 
 /**
- * Turn tables to DIV.
- * @param {object} $table Table element
+ * Converts tables that have a single `th` element with text content inside
+ * to a namespaced `div` wrapper that acts like a pseudo-component
  */
-function tableToDivs($table) {
-  const $rows = $table.querySelectorAll('tbody tr');
-  const blockname = $table.querySelector('th').textContent;
-  const $block = createTag('div', { class: `${toClassName(blockname)}` });
-  $rows.forEach(($tr) => {
-    const $row = createTag('div');
-    $tr.querySelectorAll('td').forEach(($td) => {
-      const $div = createTag('div');
-      $div.innerHTML = $td.innerHTML;
-      $row.append($div);
+function convertTables() {
```

Comment:
Can we keep the name `decorateTables()`? In Helix, when we talk about functions that manipulate the original DOM, the term decorator has been established across several projects. Those are potential candidates to be moved to the server side at some point. Keeping the naming consistent will make it easier to scan projects for impact analysis.

Reply:
hey @rofe - the method will replace tables with nested divs, so all table elements are removed from the DOM. I agree we should keep naming consistent, but we also have to think about semantics: I would expect that `decorateTables` would add some styles or classes to existing table elements.
cc @overmyheadandbody

Reply:
@narcis-radu Agreed on semantics. Let's imagine Helix "decorators" as very opinionated interior decorators who decide which pieces of furniture get to stay and which don't 😄

Reply:
Although I agree with @narcis-radu on semantics, I can see why you want to keep it the same across projects for now. I will update it today.

In regards to the server-side move of certain methods, I would like to highlight a couple of things that we currently do on top of the original method:
* transform only tables that have a single `th` element with authored content. This means that we potentially also support regular tables, as long as they have at least two columns, each with `th` content;
* only populated `table` elements are converted to `div` sections. If a `td` has no content, it will not be exposed in the resulting markup. If a `tr` has only empty `td` elements, the `tr` will be omitted from the final markup;

As an FYI, if you'd be interested in seeing the conversion time for each table, you can run `window.performance.getEntriesByType('measure')` in the console and see all the tables that have been transformed, along with the conversion time, in `ms`. If I remember correctly, these values also get exposed to Lighthouse.